metadata.json

```diff
@@ -1,5 +1,5 @@
 {
   "correlationId": "595D6107-E618-4BD9-85A0-C323EFC400D1",
   "owner":"miengine",
-  "version": "16.5"
+  "version": "17.5"
```

Comment:
>17.5 [](http://example.com/codeflow?start=14&length=4)

nit: This file could probably be deleted, but this version number is wrong. Should be '17.0'.